### PR TITLE
UXD-305 Adding types handler for react-docgen

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
Adding types handler for react-docgen

### Notes ✏️

Added and extracted functions into `parseFileToReactDoc.js` .  Script will take the `types.propName.value` from `propTypes` and `defaultProps`, then convert them into `ComponentName.types.propName.value` by using `displayName`. By doing this, we don't need to create any fake wrapper for child component just for the constants types.

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [x] PATCH (bug fix) change to `DataField` `NotificationCard` `Radio`

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: scripts/parseFileToReactDoc.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        return { ...propValue };
+      });
+    }
+
+    if (value.defaultValue && new RegExp(/^types./).test(value.defaultValue.value)) {
+      documentation._props.set(key, {
+        ...value,
+        type: {
+          ...value.type,
+          value: newValue,
```
Comment (nahumzs) [inline scripts/parseFileToReactDoc.js:24]: I'm not sure when the following could happen but:

wouldn't be better to newValue be set to `undefined` if you set it to null, the defaultValue will be `null` and be pass-down into another component, but if it's undefined the prop will not propagate.

File: scripts/readmeFiles.js [hunk 2]
```diff
@@ -158,6 +155,11 @@ const processPropTables = ({ info, path, paprikaDocs = null }) => {
           arrayOfComponentsDefinitions,
         });
 
+        if (subComponent === "Image") {
```
Comment (nahumzs) [inline scripts/readmeFiles.js]: I think we can delete this?

File: scripts/parseFileToReactDoc.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const reactDocs = require("react-docgen");
```
Comment (nahumzs) [inline scripts/parseFileToReactDoc.js:2]: I didn't know you can extend react-docgen, awesome :) 👏

File: scripts/parseFileToReactDoc.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+function typesHandler(documentation) {
+  const componentName = documentation._data.get("displayName").split(".")[0];
+
+  documentation._props.forEach((value, key) => {
+    let newValue = null;
+
+    if (value.type && value.type.name === "enum") {
+      newValue = value.type.value.map(propValue => {
+        if (new RegExp(/^types./).test(propValue.value)) {
```
Comment (alexzherdev) [inline scripts/parseFileToReactDoc.js:12]:
```suggestion
        if (/^types\./.test(propValue.value)) {
```

**Review by nahumzs — APPROVED** [review]

**Pull request merged**